open_instruct/utils.py

```diff
@@ -704,7 +763,7 @@ def upload_metadata_to_hf(
     # about a model for leaderboard displays.
     with open("tmp.json", "w") as f:
         json.dump(metadata_dict, f)
-    api = HfApi(token=os.getenv("HF_TOKEN", None))
+    api = HfApi()
```

Comment:
<img width="991" alt="image" src="https://github.com/user-attachments/assets/97688ed2-4d2d-4136-98f6-681b2cb1cffb">

HF_TOKEN already overrides, so there is no need to pass explicitly